lib/matplotlib/offsetbox.py

```diff
@@ -1270,8 +1275,24 @@ def __init__(self, offsetbox, xy,
             the offset box w.r.t. the *boxcoords*.
             The lower-left corner is (0, 0) and upper-right corner is (1, 1).
 
+        bboxprops : dict, optional
+            A dictionary of properties to set for the annotation bounding box,
+            for example *boxstyle* and *alpha*.  See `.FancyBboxPatch` for
+            details.
+
+        arrowprops: dict, optional
+            Arrow properties, see `.Annotation` for description.
+
+        fontsize: float or str, optional
+            Translated to points and passed as *mutation_scale* into
+            `.FancyBboxPatch` to scale attributes of the box style (e.g. pad
+            or rounding_size).  Passed here as *fontsize* for the convenience
+            of using the named sizes.  If not given, :rc:`legend.fontsize` is
+            used.  See `~.Text.set_fontsize` for valid values.
+
         **kwargs
-            Other parameters are identical to `.Annotation`.
+            Other `AnnotationBbox` properties.  See `~.AnnotationBbox.set` for
```

Comment:
```suggestion
            Other `AnnotationBbox` properties.  See `.AnnotationBbox.set` for
```
Minor, but I think it's slightly better to display the class here as well.